unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/LoadableShape.cs

```diff
@@ -151,7 +151,14 @@ protected virtual void AttachShape(IDCLEntity entity)
             ContentProvider provider = null;
 
             if (!string.IsNullOrEmpty(model.assetId))
+            {
                 provider = AssetCatalogBridge.i.GetContentProviderForAssetIdInSceneObjectCatalog(model.assetId);
+                if (string.IsNullOrEmpty(model.src))
+                {
+                    SceneObject sceneObject = AssetCatalogBridge.i.GetSceneObjectById(model.assetId);
+                    model.src = sceneObject.model;
+                }
```

Comment:
This is done in order to take into account the builder compatibility, since the builder already have the assets it doesn't send the src field, so we set it in case that we don't receive it